samples/cpp/benchmark_app/inputs_filling.cpp

```diff
@@ -315,13 +531,34 @@ ov::Tensor get_binary_tensor(const std::vector<std::string>& files,
                                                   inputInfo.second,
                                                   inputInfo.first,
                                                   filenames_used);
-    } else if ((type == ov::element::u8) || (type == ov::element::boolean)) {
```

Comment:
Why boolean was removed?

Reply:
Great catch, it's an artifact from the time I added support for other types of data. I've added boolean support back!